### PR TITLE
feat: Adding Post button and snackbar UI [MOSOWEB-24]

### PR DESCRIPTION
**Pull request by gvn**
https://mozilla-hub.atlassian.net/browse/MOSOWEB-24

### CODE REVIEW

File: components/nav/NavSideItem.vue [hunk 1]
```diff
@@ -75,6 +78,41 @@ const noUserVisual = computed(() => isHydrated.value && props.userOnly && !curre
...
+    }
+  }
+
+  @media screen and ( min-width: 1280px ) {
+    .lozenge.item {
+      padding-top: 12px;
+      padding-bottom: 12px;
+    }
+  }
+
```
Comment (anthony-liddle) [inline components/nav/NavSideItem.vue:115]: Just confirming on colors; are we going with the purple regardless of what they have the accent color set as?

image

Comment (wtfluckey) [reply components/nav/NavSideItem.vue:115]: ++ we should definitely use the accent color

**Comment by wtfluckey** [comment]
This is nitpicky, but in light mode when I hover right outside the `+ Post` button, it applies a hover effect that makes the button almost disappear. Doesn't affect dark mode though.

light-button-hover

File: components/publish/PublishWidget.vue [hunk 2]
```diff
@@ -504,6 +504,21 @@ onDeactivated(() => {
...
+
+    &:hover {
+      background:#7542E5;
+      color: #fff;
+    }
+  }
+  .dark .publish-button {
+    background: #CB9EFF;
+    color: #000;
+  }
```
Comment (wtfluckey) [inline components/publish/PublishWidget.vue:520]: If we use color variables for these, we shouldn't have to target the dark theme specifically

File: components/nav/NavSideItem.vue (hunk 1 above)
Comment (gvn) [reply components/nav/NavSideItem.vue:115]: This was the color that was specifically requested by design feedback but I can have that conversation. 

What is the color value that y'all would like me to use instead?

File: components/publish/PublishWidget.vue (hunk 2 above)
Comment (gvn) [reply components/publish/PublishWidget.vue:520]: I agree...just trying to bring over specific values from Figma per design request without changing other parts of the app. I can try to convince them to leave the current color values as is.

File: components/nav/NavSideItem.vue (hunk 1 above)
Comment (gvn) [reply components/nav/NavSideItem.vue:115]: Assuming `--c-primary` ?

**Comment by gvn** [comment]
> This is nitpicky, but in light mode when I hover right outside the `+ Post` button, it applies a hover effect that makes the button almost disappear. Doesn't affect dark mode though.

Fixed!

**Review by wtfluckey — APPROVED** [review]

File: components/publish/PublishWidget.vue [hunk 3]
```diff
@@ -504,6 +504,15 @@ onDeactivated(() => {
 </template>
 
 <style scoped>
+  .publish-button {
+    background: var(--c-primary);
```
Comment (jpezninjo) [inline components/publish/PublishWidget.vue:508]: :rocket:

File: locales/en.json [hunk 4]
```diff
@@ -53,7 +53,7 @@
     "clear_publish_failed": "Clear publish errors",
     "clear_upload_failed": "Clear file upload errors",
     "close": "Close",
-    "compose": "Compose",
+    "compose": "Post",
```
Comment (jpezninjo) [inline locales/en.json:56]: lol kinda funny you also added the Post entry below, too

**Review by jpezninjo — APPROVED** [review]
lgtm!

**Review by anthony-liddle — APPROVED** [review]

**Pull request merged**